sts-mock/jwks-helper-script/jwksHelperScript.mjs

```diff
@@ -9,18 +8,22 @@ const keyPair = await generateKeyPair("ES256", {
     extractable: true,
 });
 
-const privateKey = await exportJWK(keyPair.privateKey);
-writeFileSync(privateKeyFileName, JSON.stringify(privateKey), "utf8");
+let privateKey = await exportJWK(keyPair.privateKey);
+const thumbprint = await calculateJwkThumbprint(privateKey)
+privateKey.kid = thumbprint;
 
 const publicKey = await exportJWK(keyPair.publicKey);
-const jwks = {
+
+const publicKeyJwks = {
     keys: [
         {
             ...publicKey,
-            kid: randomUUID(),
+            kid: thumbprint,
```

Comment:
Using the actual key ID instead of a random UUID. I should have done this in the first place!